Automattic/Simplecons
# Update icon alignments

The new-note icon alignment was updated in #11 and collaborate alignment was updated in #12 

This is a PR to apply the same changed to the PDF version of those icons.

### To test
- Verify that collaborate.svg and collaborate.pdf are visually identical
- Verify that new-note.svg and new-note.pdf are visually identical
